### DISCUSSION TITLE
Kannada Language Leader

### DISCUSSION THREAD

**Discussion by thejeshgn** [OHF-Voice/intents#431] [Language Leaders]
I would like to volunteer.

**Comment by synesthesiam** [comment]
Great, thank you! I've added empty Kannada (`kn`) templates to the repository. I'd start with `_common.yaml` and the `homeassistant_*.yaml` files in `sentences/kn` :)

**Comment by synesthesiam** [comment]
@thejeshgn are you interested in being a language leader for Kannada?

**Reply by thejeshgn** [reply]
Yes, for sure. 

I should be able to send my first PR this weekend.
